dll/win32/shell32/wine/classes.c

```diff
@@ -44,7 +44,35 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
-#define MAX_EXTENSION_LENGTH 20
+#define MAX_EXTENSION_LENGTH 20 // FIXME: The limit is 254?
```

Comment:
Are these values mentioned somewhere on msdn?

Reply:
Not that I know. The maximum filename length is usually 255. Then one could argue if ".foo" is a file named ".foo" or a file with no name and the ".foo" extension. The limit would then be 253 or 254. Then we would have to check if extensions this long are supported everywhere.

My Windows 10 machine already has .ms-windows-store-license, .ms-lockscreencomponent-primary, .phone-companion-dummy-extension and .c5e2524a-ea46-4f67-841f-6a9465d9d515 so the limit is clearly not 20 today even if it was so in the past (unlikely).